### ISSUE TITLE
[Suggestion] toggle all off option

### ISSUE THREAD

**Issue by MedBooster** [BlueGreenMagick/Edit-Field-During-Review-Cloze#89]
![image](https://user-images.githubusercontent.com/95221545/184877847-01dd912b-1e75-4043-99df-b965b02caa39.png)
You are able to toggle all fields on, but it seems like you're not able to toggle all fields off by pressing a button, but that be possible to implement? 
I think that would be quite helpful, as you sometimes want to every field in every note type editable, but what if there's a few note-types you would want to remove it for for every field, not having to manully click every single field.

**Comment by ballin4dapandas** [comment]
yeah i accidentally toggled everything on and now i have to uncheck everything to toggle it off. it is quite painful lol
